src/module/util.test.js

```diff
@@ -0,0 +1,15 @@
+import {
+  addTask,
+} from './util.js';
+
+describe('add and remove items', () => {
+  test('add an item to the list', () => {
+    const taskList = [];
+    document.body.innerHTML = `
+    <div class='task-container'></div>
+    `;
+    addTask(taskList, localStorage);
+    const tasks = document.querySelectorAll('.task-container .task');
+    expect(tasks).toHaveLength(1);
+  });
+});
```

Comment:
- [ ] Great job on adding test case for the `add item` function, however according to the project requirements you should include test cases for `delete item` function and most of the pure functions in your project. Please add more test cases to your project